README.md

```diff
@@ -165,6 +168,12 @@ Table of Contents
 - [**Simple Dialer**](https://github.com/SimpleMobileTools/Simple-Dialer) <sup>**[[F-Droid](https://f-droid.org/app/com.simplemobiletools.dialer)]**</sup>
 - [**Koler**](https://github.com/Chooloo/koler) <sup>**[[F-Droid](https://f-droid.org/app/com.chooloo.www.koler)]**</sup>
 
+### • Dictionary
+- [**Collins Dictionary**](https://github.com/Konyaco/CollinsDictionary)
```

Comment:
rfp in progress: https://github.com/Konyaco/CollinsDictionary/issues/4 / https://gitlab.com/fdroid/rfp/-/issues/1999 :+1:

Reply:
Now live on F-Droid : https://www.f-droid.org/packages/me.konyaco.collinsdictionary/
((It will take some time to show new apps on F-Droid's website))